docs/introduction/index.md

```diff
@@ -0,0 +1,135 @@
+# First steps
+
+## Which API should I use?
+
+In shellfish, you have two different ways of doing things:
+
+- First, if you prefer a more concise, fluent syntax, use the extension methods (e.g. `path.read()`) by importing the `shellfish.syntax.all.*` package.
```

Comment:
```suggestion
- First, if you prefer a more concise syntax, use the extension methods (e.g. `path.read()`) by importing the `shellfish.syntax.all.*` package.
```

Reply:
Not correct, but I made It when I was thinking about the Processes API as well, I'll change it to `syntax.path.*` :)